docs/api/js/accounts-l1-l2.md

```diff
@@ -179,27 +180,37 @@ async claimFailedDeposit(depositHash: BytesLike): Promise<ethers.ContractTransac
-| returns                            | `PriorityOpResponse` object.                                                                                                                                                                                                                                                                                                                                     |
+| transaction.contractAddress        | The address of the L2 contract to call.                                |                                                                                                                                                                                                                                                                                     
+| transaction.calldata               | The calldata of the call transaction. It can be encoded the same way as in Ethereum.                         |
+
+| transaction.l2GasLimit             | The `l2GasLimit` for the call.  
+                                     |      
+
+| transaction.l2Value (optional)     | The `l2Value` for the call.                                                                                                                                                                                                           |
+| transaction.factoryDeps (optional) | Array of bytecodes of factory dependencies - only used for transactions that deploy contracts.                                                                                                                       
+| transaction.operatorTip (optional) | If the ETH `value` passed with the transaction is not explicitly stated in the overrides, this field will be equal to the tip the operator will receive on top of the base cost of the transaction. This value has no meaning for the `Deque` type of queue, but it will be used to prioritize the transactions that get into the `Heap` or `HeapBuffer` queues. 
+                                                                                                        |
+| overrides (optional)               | Ethereum transaction overrides. May be used to pass `gasLimit`, `gasPrice` etc.
+                                |
+| transaction.gasPerPubdataByte (optional)      |
+                              | 
+| transaction.refundRecipient(optional)         |  
+|--                                                                                                                                                         
+| returns                               | `PriorityOpResponse` object.                                                                                                                                                                                                                                                                                                                                     
```

Comment:
@krofax  I updated the context of the table, but it is displayed very bed in the preview for me...could you please correct it

Reply:
fixed